tests/unit/client_tests.py

```diff
@@ -539,6 +539,38 @@ def test_constructor_with_account(self):
             'https://{0}.cloudant.com'.format(self.account)
             )
 
+    def test_bluemix_constructor(self):
+        """
+        Test instantiating a client object using a VCAP_SERVICES environment
+        variable.
+        """
+        instance_name = 'Cloudant NoSQL DB-lv'
+        vcap_services = {'cloudantNoSQLDB': [{
```

Comment:
I think it would be good to test with a JSON object that has multiple Cloudant instances to verify that `instance_name` is working correctly.